### PR TITLE
added landing

### PR DESCRIPTION
**Pull request by MichaelSdet**
here is the first landing page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<body>
+    <header class="header">header</header>
+    <main class="main">
+        <div class="div">card 1</div>
+        <div class="div">card 2</div>
+        <div class="div">card 3</div>
+    </main>
+    <footer class="footer">footer</footer>
+</body>
+</html>
```
Comment (Nazarii98) [inline index.html]:
```suggestion
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Document</title>
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <header class="header">header</header>
        <main class="main">
            <div class="div">card 1</div>
            <div class="div">card 2</div>
            <div class="div">card 3</div>
        </main>
        <footer class="footer">footer</footer>
    </body>
</html>
```

**Review by Nazarii98 — CHANGES_REQUESTED** [review]
Fix one linter problem, also change title of your page to 'Michal the best'

**Review by MichaelSdet — COMMENTED** [review]
changed the title and fixed line problem

**Review by Nazarii98 — APPROVED** [review]
Great job!

**Pull request merged**